app/components/collection_metadata_component.rb

```diff
@@ -31,11 +31,17 @@ class CollectionMetadataComponent < ActionView::Component::Base
   ].freeze
 
   def initialize(collection:)
-    @collection = collection
+    @collection = decorate(collection)
   end
 
   private
 
+    def decorate(col)
+      return col if col.is_a? ResourceDecorator
+
+      ResourceDecorator.new(col)
```

Comment:
Can we just make all the collection displays use the ResourceDecorator? We're already doing it in the resources show page. Is the only other place is the dashboard view?